index.md

```diff
@@ -66,25 +66,21 @@ Not sure where to start? Check out [our resources](#resources).
+* [Project Include](http://projectinclude.org/), a community for building meaningful, enduring diversity and inclusion into tech companies, has a number of templates available to use
+* [Venture Inclusion Network](https://www.ventureinclusion.com/), hosts a curated directory of experts and services, initiatives in the industry, sample policies, best practices and more
 * [A template for investor/founder sexual harassment policy](https://techcrunch.com/2017/07/05/harassment-policy-template/) (TechCrunch)
 * [Level classifications of harassment and assault](https://cherylyeoh.com/2017/07/03/shedding-light-on-the-black-box-of-inappropriateness/) (Cheryl Yeoh Sew Hoy)
 * [Anti-Harassment Policy Template for Startups](https://news.greylock.com/anti-harassment-policy-template-for-startups-fe24a580bddf) (Greylock Partners)
 * [Staying Safe When You Say #MeToo](https://www.aclu.org/blog/privacy-technology/internet-privacy/staying-safe-when-you-say-metoo) (ALCU)
 
-**Organizations, platforms, and tools:**
-* [Project Include](http://projectinclude.org/), a community for building meaningful, enduring diversity and inclusion into tech companies
-* [Callisto](https://www.projectcallisto.org/), an online platform that detects serial perpetrators of sexual assault and professional sexual coercion
-* [Paradigm](https://www.paradigmiq.com/), a diversity and inclusion strategy firm that partners with leading technology startups and Fortune 500 companies to build stronger, more inclusive organizations
-* [Forshay](http://forshay.com/diversity-inclusion/), a strategy firm that works with companies to design D&I strategy, deliver training, and provide consulting services
-* [Inclusion Ventures](http://inclusionventures.com/), a strategic consulting firm that provides training to help organizations and VCs take an inclusive approach to diversity and leadership
-* [FemaleFounder.org](https://www.femalefounder.org/), a community of women helping women (e.g., female founder office hours hosted across the country with top-tier female VCs)
-* [Parity.org](https://www.parity.org/), an organization that encourages corporations to commit to interviewing at least one qualified woman for all open roles at the VP level and above
-* _More coming soon._  A number of organizations have informed us of initiatives that will launch in the next few months
 
-If your organization is working on any tools or services for VCs shifting the diversity, inclusion, and anti-harassment paradigm and would like to be included on this list, please email us at _hello@venturemovingforward.org_.
+**Organizations, consultants, and tools:**
+* Check out [Venture Inclusion Network](https://www.ventureinclusion.com/experts-and-service-providers/) for a list of D&I experts, consulting firms and service providers.
+
+* _Coming Soon: We're also developing a Product Hunt Collection for tools that support diverse, inclusive, and harassment-free workplaces._
```

Comment:
Lowercase "soon" to make it sentence case as well

Reply:
+1